markdown/src/viewer.c

```diff
@@ -297,15 +297,15 @@ pop_scroll_pos(MarkdownViewer *self)
 }
 
 static void
-on_webview_load_status_notify(WebKitWebView *view, GParamSpec *pspec,
+on_webview_is_loading_notify(WebKitWebView *view, GParamSpec *pspec,
   MarkdownViewer *self)
 {
-  WebKitLoadEvent load_status;
+  gboolean load_status;
```

Comment:
Likewise, instead of using this boolean, to look for [`WEBKIT_LOAD_FINISHED`](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WEBKIT-LOAD-FINISHED:CAPS)?

Reply:
Fixed, thanks. Looks like I missed that signal while looking through the docs.